observation_portal/requestgroups/duration_utils.py

```diff
@@ -58,19 +58,21 @@ def get_configuration_duration(configuration_dict):
         conf_duration['duration'] = configuration_dict['repeat_duration']
     else:
         conf_duration['duration'] = sum([icd['duration'] for icd in instrumentconf_durations])
-    conf_duration['duration'] += (PER_CONFIGURATION_STARTUP_TIME)
+        conf_duration['duration'] += request_overheads['config_front_padding']
```

Comment:
Here we only apply the front_padding to non-repeat-expose configurations, since the repeat_duration is now expected to contain the config_front_padding within it.